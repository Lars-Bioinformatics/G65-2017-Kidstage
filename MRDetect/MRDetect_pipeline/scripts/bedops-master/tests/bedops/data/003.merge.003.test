chr6	5	11
chr6	16	17
chr6	17	118
chr6	119	121
