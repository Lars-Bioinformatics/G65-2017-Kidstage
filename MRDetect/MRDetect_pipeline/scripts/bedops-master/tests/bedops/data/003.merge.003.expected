chr6	5	11
chr6	16	118
chr6	119	121
